test/miscellaneous/odp_dyn_workers.c

```diff
@@ -971,10 +1135,30 @@ static odp_bool_t check_summary(const summary_t *summary)
 	return true;
 }
 
+static void dump_result(int socket, pid_t pid)
```

Comment:
Optional, but if it's simple to implement it could be handy to have a separate CLI command for printing active workers from all processes (e.g. `PRINT_WORKERS`).

Reply:
This would change some assumptions in the code so requires a few changes. Perhaps a future addition in case the new addition dumps are deemed inadequate for some reason.

Reply:
Okay, can be implemented in the future if deemed useful.